### DISCUSSION TITLE
Oshaveryのライセンスをどうするか

### DISCUSSION THREAD

**Discussion by YukiYuigishi** [Growthers/Oshavery#34] [Ideas]
v0.1はMITでしたが、v0.2以降はライセンスを変更しないかという話が何度か上がっています。
例を上げなるなら、laminneさんがAGPLにしないかという提案をしています。
コントリビューターのみなさんは少なくとも１度は意見を表明していただけると助かります。

**Comment by laminne** [comment]
AGPLはWeb経由であっても適用されるGPLライセンスで,フォークを本家が取り込むこともできるライセンスです(なのでAGPLにしておきたい
> 派生物のライセンスが元のコードと同一であるため、世界中の誰が行った改変であれ、コアに取り込むことができます。他の人にも役立つ改変がコアに取り込まれることでソフトウェア自体が改善されます。ソースコードが共有されるため、開発効率も良くなります(車輪の再発明を極力減らすことができる)。
https://ounziw.com/2013/05/17/agpl/

**Reply by YukiYuigishi** [reply]
車輪の再発明を減らせるのはでかいな

**Reply by re-taro** [reply]
初めてこのライセンスの中身覗いたけど、すごい良い

**Reply by Yusuke4869** [reply]
いいと思います

**Reply by Sigumaa** [reply]
良いと思います
